adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -184,6 +184,58 @@ def detect_taps(self, value):
                     value, 60, time_limit=10, time_latency=50, time_window=255
                 )
 
+    # pylint: disable-msg=too-many-arguments
+    # Many default arguments which will probably not need changing in most cases
+    def configure_taps(
+        self,
+        value,
+        accel_range=adafruit_lis3dh.RANGE_8_G,
+        theshold=None,
+        time_limit=None,
+        time_latency=50,
+        time_window=255,
+    ):
+        """ A way to customize the tap detection better. The default values don't work
+            for all cases.
+            Higher default thresholds for the CPB
+        """
+        if value < 0 or value > 2:
+            return
+
+        if accel_range not in [
+            adafruit_lis3dh.RANGE_2_G,
+            adafruit_lis3dh.RANGE_4_G,
+            adafruit_lis3dh.RANGE_8_G,
+            adafruit_lis3dh.RANGE_16_G,
+        ]:
+            accel_range = adafruit_lis3dh.RANGE_8_G
+        self._lis3dh.range = accel_range
+
+        if value == 1:
+            if theshold is None or theshold < 0 or theshold > 127:
+                theshold = 100 if "nRF52840" in os.uname().machine else 90
+            if time_limit is None:
+                time_limit = 4
+        elif value == 2:
+            if theshold is None or theshold < 0 or theshold > 127:
+                theshold = 70 if "nRF52840" in os.uname().machine else 60
+            if time_limit is None:
+                time_limit = 10
+        else:
+            # sane values for turning the tap detection off
+            theshold = 100
+            time_limit = 1
+
+        self._lis3dh.set_tap(
+            value,
+            theshold,
+            time_limit=time_limit,
+            time_latency=time_latency,
+            time_window=time_window,
+        )
+
+    # pylint: enable-msg=too-many-arguments
```

Comment:
This can be removed if you update the `disable` as suggested above.

Reply:
Cleaned up.